cli/src/main/java/com/devonfw/tools/ide/commandlet/UpgradeSettingsCommandlet.java

```diff
@@ -181,4 +187,43 @@ private static void updatePropertiesLegacyEdition(EnvironmentVariablesProperties
       environmentVariables.remove(legacyEditionVariable);
     }
   }
+
+  private void cleanupLegacyProperties() {
+    this.context.info("Cleaning up legacy properties...");
+
+    Path rootDirectory = Paths.get(System.getProperty("user.dir"));
+    try {
+      Files.walk(rootDirectory)
+          .filter(path -> path.getFileName().toString().equals("IDEasy.properties"))
```

Comment:
Obsolete but for the record. This filename `IDEasy.properties` is just an example and cannot be hardcoded.
There can be any number of `*.properites` files in the `repositories` folder and you cannot make any assumptions about their name.